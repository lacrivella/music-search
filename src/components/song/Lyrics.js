import React from 'react';
import PropTypes from 'prop-types';

function Lyrics({ artist, title, lyrics }) {
  return (
    <section>
      <h3>{title}</h3>
      <p>by {artist}</p>
      <p>{lyrics ? lyrics : 'This song suxs it has no lyrics!'}</p>
    </section>
  );
}

Lyrics.propTypes = {
  artist: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  lyrics: PropTypes.string.isRequired
};

export default Lyrics;
