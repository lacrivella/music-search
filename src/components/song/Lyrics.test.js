import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics';

describe('Lyrics component', () => {
  it('renders Lyrics', () => {
    const wrapper = shallow(<Lyrics artist="Cat" title="Meow" lyrics="meow meow meow meeeow"/>);
    expect(wrapper).toMatchSnapshot();
  });
});