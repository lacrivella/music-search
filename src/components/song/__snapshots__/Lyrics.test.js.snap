// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders Lyrics 1`] = `
<section
  className="Lyrics"
>
  <h3>
    Meow
  </h3>
  <p>
    by 
    Cat
  </p>
  <p>
    meow meow meow meeeow
  </p>
</section>
`;
